src/js/components/Collapsible.js

```diff
@@ -0,0 +1,66 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import TransitionGroup from 'react-addons-transition-group';
```

Comment:
I'm assuming we are using transition group directly only because Animate component was not approved yet. Is this interpretation correct?